README.md

```diff
@@ -47,8 +47,8 @@ Components in beta stage may include breaking changes, new features, and bug fix
 |--------------------|---------|--------|
 | [terra-alert](https://github.com/cerner/terra-core/tree/master/packages/terra-alert) | [![NPM version](http://img.shields.io/npm/v/terra-alert.svg)](https://www.npmjs.org/package/terra-alert) | :white_check_mark: |
 | [terra-arrange](https://github.com/cerner/terra-core/tree/master/packages/terra-arrange) | [![NPM version](http://img.shields.io/npm/v/terra-arrange.svg)](https://www.npmjs.org/package/terra-arrange) | :white_check_mark: |
-| [terra-app-delegate](https://github.com/cerner/terra-core/tree/master/packages/terra-app-delegate)  | [![NPM version](http://img.shields.io/npm/v/terra-app-delegate.svg)](https://www.npmjs.org/package/terra-app-delegate) | :white_check_mark: |
-| terra-application  | [![NPM version](http://img.shields.io/npm/v/terra-application.svg)](https://www.npmjs.org/package/terra-application) | :x: |
+| terra-app-delegate | [![NPM version](http://img.shields.io/npm/v/terra-app-delegate.svg)](https://www.npmjs.org/package/terra-app-delegate) | :x: |
```

Comment:
I vote we just remove reference from the readme (or comment that the package moved), the package isn't actually deprecated, just moved.

Reply:
Yeah, I think we just remove the references from this README file.